### PR TITLE
Remove incorrect information from Studio update sites articles

### PR DESCRIPTION
**Pull request by fermujica**

### CODE REVIEW

**Review by agustincelentano — APPROVED** [review]

File: modules/ROOT/pages/studio-update-sites.adoc [hunk 1]
```diff
@@ -4,86 +4,34 @@ include::_attributes.adoc[]
 endif::[]
 :keywords: mule, esb, studio, anypoint studio, update sites, updates, extensions, plug-ins, plugins, download
 
-Update sites in Anypoint Studio are used to update or install Studio, the embedded Mule runtime, connectors or other useful development tools.
+Anypoint Studio uses update sites to update Studio, and the embedded versions of Mule runtime engine and MUnit.
```
Comment (kbostrom) [inline modules/ROOT/pages/studio-update-sites.adoc]: No comma; the comma could be interpreted to mean that Studio uses them to update itself...lol....
```suggestion
Anypoint Studio uses update sites to update both itself and embedded versions of Mule runtime engine and MUnit.
```

File: modules/ROOT/pages/studio-update-sites.adoc [hunk 2]
```diff
@@ -4,86 +4,34 @@ include::_attributes.adoc[]
...
 
 == Accesing an Update Site
 
-. Go to *Help* > *Install New Software*.
-. Select one of these update sites from the drop-down menu next to the *Work with* field.
+. Navigate to *Help* > *Install New Software*.
+. In the *Work with* field, select  one of the update sites.
```
Comment (kbostrom) [inline modules/ROOT/pages/studio-update-sites.adoc]: omit space:
```suggestion
. In the *Work with* field, select one of the update sites.
```

File: modules/ROOT/pages/studio-update-sites.adoc [hunk 3]
```diff
@@ -4,86 +4,34 @@ include::_attributes.adoc[]
...
-Updates the RAML Editor in Anypoint Studio.
-
-`+http://studio.mulesoft.org.s3.amazonaws.com/s2/ramleditor/+`
-
-== MUnit Update Site
-
-xref:2.1@munit::index.adoc[MUnit] framework for testing Mule Applications
-
-`+http://studio.mulesoft.org/s2/munit+`
+To add a new update site to install extra components to Studio:
```
Comment (kbostrom) [inline modules/ROOT/pages/studio-update-sites.adoc]: Strictly optional:
```suggestion
To add a new update site for installing additional components to Studio:
```

File: modules/ROOT/pages/studio-update-sites.adoc [hunk 4]
```diff
@@ -4,86 +4,34 @@ include::_attributes.adoc[]
...
 
+. Navigate to *Help* > *Install New Software*.
+. Click *Add...*.
+. In *Name*, add a to identify the update site.
```
Comment (kbostrom) [inline modules/ROOT/pages/studio-update-sites.adoc]: add a name? or something more fancy?
```suggestion
. In *Name*, add a name to identify the update site.
```

File: modules/ROOT/pages/studio-update-sites.adoc [hunk 5]
```diff
@@ -4,86 +4,34 @@ include::_attributes.adoc[]
...
-* `+http://repository.mulesoft.org/connectors/releases/+`
-
-* `+http://download.eclipse.org/technology/m2e/releases+`
-
-* `+http://download.eclipse.org/eclipse/updates+`
-
-* `+http://subclipse.tigris.org/+`
-
-* `+http://findbugs.cs.umd.edu/eclipse/+`
+If you are on a network with firewall restrictions, you must whitelist the URLs of the update sites to download extensions:
```
Comment (kbostrom) [inline modules/ROOT/pages/studio-update-sites.adoc]:
```suggestion
If you are on a network that has firewall restrictions, you must whitelist the URLs of the update sites to download extensions:
```

File: modules/ROOT/pages/studio-update-sites.adoc [hunk 6]
```diff
@@ -4,86 +4,34 @@ include::_attributes.adoc[]
...
 
+. Navigate to *Help* > *Install New Software*.
+. Click *Manage...*.
+. The list of available software shows up:
```
Comment (kbostrom) [inline modules/ROOT/pages/studio-update-sites.adoc]: Not a step
```suggestion
The list of available software is displayed:
```

**Review by kbostrom — APPROVED** [review]
Here you go!
Please let me know if you have questions or comments, and please do let me know if my suggested rewordings change the technical meaning, so that I can learn more!  :-)

**Pull request merged**